account_tax_cash_basis_reference/models/account_move.py

```diff
@@ -27,6 +27,16 @@ def _compute_nbr_moves(self):
                 rec.nbr_moves = len(rec.account_move_ids)
 
     @api.multi
+    def _reverse_move(self, date=None, journal_id=None):
+        if self.tax_cash_basis_rec_id:
+            date = (
+                self.tax_cash_basis_rec_id.debit_move_id.date if
+                self.tax_cash_basis_rec_id.debit_move_id
+                .journal_id.type == 'bank' else
```

Comment:
`in ['bank', 'cash'] ` to allow cash payments too.

Reply:
@alan196 the change is done. Please check it :pray: